cloudferry/config.py

```diff
@@ -289,6 +289,8 @@ class DatabaseSettings(ConfigSection):
 
 class OpenstackCloud(ConfigSection):
     name = fields.String()
+    request_attempts = fields.Integer(missing=3)
```

Comment:
No, I can't. I still need to figure out how to generate configuration documentation from config, so It is separate task.